django/pagai/api/views.py

```diff
@@ -40,28 +40,28 @@ def post(self, request, owner):
         return Response(db_schema, status=status.HTTP_200_OK)
 
 
-class ExploreView(views.APIView):
-    def get_serializer_class(self):
-        return ExplorationSerializer
+@extend_schema(
+    request=serializers.ExplorationRequestSerializer, responses={"201": serializers.ExplorationResponseSerializer}
+)
+class ExploreView(generics.CreateAPIView):
+    def create(self, request, *args, **kwargs):
```

Comment:
Does this mean we'll have an Explorations table storing all the explorations that have been made?

Reply:
no no we don't. it's just that I had to use the DRF generics views to have a proper openapi schema. The decoractor `@extend_schema` didn't mix well with simple `APIView` for some reason.